src/app/dynamic-forms/models/cinchy-form.model.ts

```diff
@@ -874,59 +872,59 @@ export class Form {
         if (field.cinchyColumn.name) {
           allRowData.forEach((rowData: { [key: string]: any }) => {
 
-          //TODO: Passing array value in case of multiselect
-          if (field.cinchyColumn.dataType === "Choice" && field.cinchyColumn.isMultiple === true) {
-            const valueArray = rowData[field.cinchyColumn.name]?.split(",") ?? [];
+            //TODO: Passing array value in case of multiselect
```

Comment:
Do you really want to remove this for the code which I have not written.  I hardly had a change in that file and the changes are showing because my editor changed the format of the file.

Reply:
done